### PR TITLE
README.md: update the README of the documentation repo

### PR DESCRIPTION
**Pull request by gnawux**
Ship more information of the documents in the README page.

Fixes: #244

### CODE REVIEW

**Comment by gnawux** [comment]
It surprised me that a PR in the `documentation` repo could trigger our CI... 😂

**Comment by jodh-intel** [comment]
哈哈 :smile:

lgtm!

File: README.md [hunk 1]
```diff
@@ -7,17 +7,44 @@ common to multiple components.
...
+
 * [Release strategy](Releases.md)
 * [Release Checklist](Release-Checklist.md)
-* [Upgrading](Upgrading.md)
-* [OpenStack Zun with Kata Containers](zun/zun_kata.md)
+
+## Help Improving the Documents
+
+* [Documentation Requirements](Documentation-Requirements.md)
+* [Contributing](CONTRIBUTING.md)
```
Comment (grahamwhaley) [inline README.md]: Maybe the Contributing and CoC entries should move up to the Contributing section?

**Comment by grahamwhaley** [comment]
Yeah, @jodh-intel mentioned recently that maybe we can make the CI spot any diffsets that only contain docs changes and do a very minimal CI (I believe we scan the docs to see if they have executable bash sections in some cases and test-run those sections). The CI also does the SoB check etc. So, maybe we cannot do away with the CI on this repo, but we can surely optimise it :-)

File: README.md (hunk 1 above)
Comment (grahamwhaley) [reply README.md]: oh, they don't 301 for me - if you look on the 'files changed' tab of this PR, and then open the 'view' button (which views your modified file in your branch), then those contrib and CoC links work fine for me :-)

Sure, drop the extra CONTRIBUTING. I think I'd still move the CoC up into the contributing section.

Comment (gnawux) [reply README.md]: OK for me. They are a bit like when I unboxed a gift package and got a smaller package 😄

**Pull request merged**